backend/db/models/Question.js

```diff
@@ -0,0 +1,18 @@
+const connection = require('../connection')
+;
+const { Sequelize } = connection;
+const { UUID, UUIDV4, TEXT, STRING } = Sequelize;
+
+module.exports = connection.define('question', {
+  id: {
+    type: UUID,
+    primaryKey: true,
+    defaultValue: UUIDV4
+  },
+  text: {
+    type: TEXT
+  },
+  functionName: {
+    type: STRING
+  }
```

Comment:
Could we add add a `params` property? It'll help with displaying the function signature on the front end.